examples/ChatQnA/deploy/monitoring.md

```diff
@@ -0,0 +1,133 @@
+
+# Monitoring ChatQnA deployment
+
+Now that you have deployed the ChatQnA example, let's talk about monitoring the performance of the microservices in the ChatQnA pipeline.
+
+Monitoring the performance of microservices is crucial for ensuring the smooth operation of the generative AI systems. By monitoring metrics such as latency and throughput, you can identify bottlenecks, detect anomalies, and optimize the performance of individual microservices. This allows us to proactively address any issues and ensure that the ChatQnA pipeline is running efficiently.
+
+This document will help you understand how to monitor in real time the latency, throughput, and other metrics of different microservices. You will use **Prometheus** and **Grafana**, both open-source toolkits, to collect metrics and visualize them in a dashboard.
+
+### Set up the Prometheus server
```

Comment:
You shouldn't skip from the H1 (title) to an H3 heading.  This should be an H2 heading (with two leading ##).